### PR TITLE
Update to 3.0 stable

### PR DESCRIPTION
**Pull request by vinniefranco**
### What's new?

3.0 support! AND we got to delete some code for the same functionality!

### What's on the horizon?

* https://github.com/spree/spree/pull/6627

Soon HATBM model will be dead, and the majority of this gems function will not be needed.

### CODE REVIEW

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ruby:
+  enabled: true
+  config_file: .rubocop.yml
+
+scss:
+  config_file: .scss-style.yml
```
Comment (sikachu) [inline .hound.yml]: Can you fix your editor so there's a newline at the end of file?

**Comment by sikachu** [comment]
Just one minor thing. Anything else looks fine. :ship: :it:

**Comment by BenMorganIO** [comment]
Happy my late night insomnia and making the bad decision to turn [f.lux](https://justgetflux.com/) off actually helped someone :D

**Comment by vinniefranco** [comment]
@BenMorganIO thanks for your work!

**Comment by BenMorganIO** [comment]
@vinniefranco Getting that work into [Solidus](https://github.com/solidusio/solidus) as well; the popular Spree fork.

**Pull request merged**